web/src/routes/OrganizationsList/Organization/Tabs/TeamsAndMembership/TeamsView/ManageMembers/ManageMembersList.tsx

```diff
@@ -105,6 +119,15 @@ export default function ManageMembersList(props: ManageMembersListProps) {
   const [isDeleteModalForRowOpen, setIsDeleteModalForRowOpen] = useState(false);
   const [memberToBeDeleted, setMemberToBeDeleted] = useState<IMemberInfo>();
 
+  // Team Sync states
+  const [isOIDCTeamSyncModalOpen, setIsOIDCTeamSyncModalOpen] = useState(false);
+  const [OIDCGroupName, setOIDCGroupName] = useState<string>('');
+  const [pageInReadOnlyMode, setPageInReadOnlyMode] = useState<boolean>(false);
+  const [isRemoveTeamSyncModalOpen, setRemoveTeamSyncModalOpen] =
+    useState<boolean>(false);
+  const [teamSyncConfigExpanded, setTeamSyncConfigExpanded] =
+    useState('oidc-config-toggle');
```

Comment:
Should this be a boolean? Or is this for implementing LDAP later?

Reply:
on a second look, it can be boolean and makes sense to have it boolean. I changed it.